src/components/shared/Page/Page.stories.js

```diff
@@ -14,7 +13,9 @@ const stories = storiesOf('Shared.Page', module);
 
 stories
   .addDecorator(withKnobs)
-  .addDecorator((story) => <LastLessonProvider>{story()}</LastLessonProvider>)
+  .addDecorator((story) => {
```

Comment:
You can delete this decorator since it's no longer doing anything

Reply:
Done!